scripts/examples/typescript/README.md

```diff
@@ -0,0 +1,41 @@
+# A Typescript bindings for XenAPI
+A Typescript bindings for XenAPI, which is inspired by [Python XenAPI](https://xapi-project.github.io/xen-api/usage.html).
+
+## Usage
+Install lib from npm repository
```

Comment:
it's not that straightforward unless you start publishing on `npmjs.com`

I'd do it this way:
1. add these bindings to the SDK build actions so they're built on every PR
2. add the NPM package as an artefact to the release GitHub action
3. Point users to the last release's artefacts and give them instructions on how to install the package using those instead of a generic `npm i`